benchmarks/buiter_et_al_2016_jsg/exp_2_high_resolution.prm

```diff
@@ -0,0 +1,208 @@
+#### This parameter file reproduces the unstable wedge experiment 2 from Buiter et al., 2016, JSG.
+###  Experiment 2 tests how an unstable subcritical wedge deforms to reach the critical taper
+###  solution. Horizontal layers of sand have 10 cm shortening by inward movement of a mobile
+###  wall with a velocity of 2.5 cm/hour. Both the base and the surface are horizontal and the 
+###  “wedge” is thus in the unstable field. This experiment builds a thrust wedge through a 
+###  combination of mainly in-sequence forward and backward thrusting. Deformation starts by 
+###  forming a pop-up structure near the high viscosity rigid indenter.
+
+###  Global parameters
+set Dimension                                    = 2
+set Start time                                   = 0
+set End time                                     = 14400  #Equivalent to ~10 cm of shortening
+set Use years in output instead of seconds       = false
+set CFL number                 = 0.5
+set Output directory                             = Brittle_thrust_wedge_exp2
+set Pressure normalization                       = no
+set Timing output frequency                      = 20
+
+## Note that the Linear/Nonlinear solver tolerance should be sufficiently strict to avoid numerical instabilities.
+set Nonlinear solver scheme                      = single Advection, iterated Stokes
+set Nonlinear solver tolerance                   = 1e-7
+set Max nonlinear iterations                     = 100
+
+subsection Solver parameters
+  subsection Stokes solver parameters
+    set Linear solver tolerance             = 1e-8    
+    set Number of cheap Stokes solver steps       = 0
+    set GMRES solver restart length              = 200   
+## a higher restart length makes the solver more robust for large viscosity contrasts.
+  end
+end
+
+# Decreasing the DoFs in Temperature and composition
+subsection Discretization
+  set Composition polynomial degree              = 2
+  set Temperature polynomial degree              = 2
+  set Stokes velocity polynomial degree          = 2
+  set Use locally conservative discretization       = false
+end
+
+# For restarting the model
+set Resume computation                           = auto
+subsection Checkpointing
+  set Steps between checkpoint             = 10
```

Comment:
That's quite frequent. How many time steps are you running in total? I would maybe checkpoint 20 or 50 times over the course of a simulation.